### DISCUSSION TITLE
Keyframes Bug

### DISCUSSION THREAD

**Discussion by nuddbudd** [keiwando/toonsquid#126] [Ideas]
While animating randomly my keyframes stopped working entirely. When I'd add a new keyframe and move it nothing would happen and it'd just move the whole thing with no movement at all, as if i didn't even use a keyframe. I used keyframes before in the same exact project but it just randomly stopped working. I waited the next day to continue the project and it still isn't working. Keyframes work on every other project expect that one for some reason. 

I couldn't find anyone struggling with the same thing so I'm making my own comment. Hopefully this is fixed soon. 

I'm putting this in the ideas category since I don't know where to put this, so apologies if it's not the write category.

**Comment by keiwando** [comment]
Hi! If you are trying to animate pixel layers with keyframes, the [Edit pixels](https://toonsquid.com/handbook/transform/transform_tool/#tool-options) setting of the transform tool has to be disabled. Maybe you accidentally enabled that without noticing?
